deploy_scripts/start_services

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-sudo chmod 777 /home/ubuntu/projects/executor/gradlew
+# sudo chmod 777 /home/ubuntu/projects/executor/gradlew
```

Comment:
Why is this change needed?

Reply:
I's now needed right now, that's why I commented out. 
At the first time I added this line because when trying to execute a test after the deploy it causes an error of permissions since the aws agent starts the deploy as _root_ and the support project starts the execution as _ubuntu_ user.

Now it's changing the owner of the executor project to _ubuntu_ in line _3_ of the _deploy_scripts/config_files_